public/components/event_analytics/utils/utils.tsx

```diff
@@ -270,6 +254,23 @@ export const getPropName = (queriedVizObj: {
   }
 };
 
+export const getMetricVisConfig = (metric) => {
+  const { aggregation } = metric;
+  return {
+    [AGGREGATIONS]: [
```

Comment:
Language/Style question: What does an object key in an array signify? Not familiar with the syntax, google is confusing my question about "array as object key" as a query about `Object.keys`.